internal/cmd/load-balancer/generate-payload/generate_payload_test.go

```diff
@@ -190,3 +192,68 @@ func TestBuildRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestToModifyListeners(t *testing.T) {
```

Comment:
```suggestion
func TestModifyListeners(t *testing.T) {
```

Reply:
Maybe the test could contain at least two listeners just to check that the name is being removed from all

Reply:
changed function name and add one more listener